### ISSUE TITLE
Update all tutorials and ZopeSkel references for Plone 4.3 

### ISSUE THREAD

**Issue by miohtama** [collective/collective.developermanual#183]
Now that 4.3 ships with ZopeSkel out of the box (thanks @smcmahon  for very hard work) we need to update all tutorials to reflect this reality and get rid of the fragile old boilerplate.
- Check which zopeskel version 4.3 comes with
- Standardize content creation to this zopeskel version
- Standardize theme creation to this zopeskel version (which is actually Diazo compatible theme package, is such needed anymore?)
- Update instructions how to export Dexterity content types to this version
- Update all tutorials or add _Dragons_ warnings if they cannot be updated and are no longer compatible

@svx could you help with this task?

@zupo what are the implications of todoapp tutorial? Last time I checked it used a custom buildout, but I would prefer to see a solution where we start from the scratch with vanilla Plone installation.

:beers:  for everyone.

**Comment by cleder** [comment]
> (which is actually Diazo compatible theme package, is such needed anymore?)

I would argue yes. In my experience it is more convenient to edit and maintain a diazo 
theme on the filesystem although I prefer to deploy them as uploadable zip files.

**Comment by aclark4life** [comment]
Wow, very exciting! Thanks @smcmahon

**Comment by smcmahon** [comment]
ZopeSkel's actually been shipping with the installers for some time, in the develop.cfg. What's new with 4.3 is that we're including zopeskel.dexterity and zopeskel.diazo.

[versions]
ZopeSkel = 2.21.2
zopeskel.dexterity = 1.5.0
zopeskel.diazotheme = 1.1

The diazotheme skeleton covers a more sophisticated need than what you can handle via zip file:

1) It includes z3c.jbot and has its own browserlayer;

2) It includes the profile machinery to add resources to portal_css and portal_javascripts.

I think an ideal theming tutorial would start TTP, then say "need more?" Create a package with zopeskel.diazo, drop the exported work so far into it, and develop on! The Dexterity tutorial could do pretty much the same.

[BTW, I know that Templer's not far from useful. But it wasn't ready for 4.3, and that's that.]

**Comment by svx** [comment]
@miohtama yes i can, but it will take some time, first i have to deal with some insane deadlines :)
